ui/page/popoutChatWrapper/view.jsx

```diff
@@ -1,10 +1,12 @@
 // @flow
 import { formatLbryChannelName } from 'util/url';
-import ChatLayout from 'component/chat';
+import { lazyImport } from 'util/lazyImport';
 import Page from 'component/page';
 import React from 'react';
 import Yrbl from 'component/yrbl';
 
+const ChatLayout = lazyImport(() => import('component/chat' /* webpackChunkName: "chat" */));
```

Comment:
There are other components where the chat is loaded conditionally, so the chunk already exists anyways and yeah, it's also required for the lazy css. Cases like this only exists when lazy imports of the components make sense in at least one place.

Reply:
For the JS side, I recall it will be embedded in ui.js when there is any instance that is not wrapped in Suspense (otherwise it doesn't make sense to be both static and lazy). 

I usually just remove the lazy loading for this component entirely all since it is not working.  If not mistaken, no js chunk will be produced, or the one in ui.js will be used (forgot which one).

Reply:
What happens now when we decide not lazy load a component?  Just rename the css file?

Reply:
(I could be wrong about my previous comment, since I was dealing with old webpack)